src/09_example/02_handling_connections_concurrently.md

```diff
@@ -36,5 +36,19 @@ Now, we can handle each connection concurrently by passing `handle_connection` i
 The closure function takes ownership of each `TcpStream`, and is run as soon as a new `TcpStream` becomes available.
 As long as `handle_connection` does not block, a slow request will no longer prevent other requests from completing.
 ```rust,ignore
-{{#include ../../examples/08_04_concurrent_tcp_server/src/main.rs:main_func}}
-```
\ No newline at end of file
+{{#include ../../examples/09_04_concurrent_tcp_server/src/main.rs:main_func}}
+```
+# Serving Requests in Parallel
+Our example so far has largely presented concurrency (using async code)
+as an alternative to parallelism (using threads).
+However, async code and threads are not mutually exclusive.
+In our example, `for_each_concurrent` processes each connection concurrently on the same thread as the `main` function.
```

Comment:
`for_each_concurrent` processes each connection as a single task (i.e. on the same thread), but which thread depends on the executor. "processes each connection concurrently, but on the same thread" would be more accurate.